src/css/main.css

```diff
@@ -138,36 +137,53 @@ a:active {
   height: 16px;
 }
 
+.incident h1 {
+  font-size: 1.5em;
+  margin-top: 0;
+}
+
 .incident {
   color: #202020;
 }
 
 .incident-title {
-  float: left;
   text-transform: capitalize;
+  width: 75%;
 }
 
-.incident-date {
-  float: right;
-  font-size: 14px;
-  opacity: 0.3;
-}
-
-@media (max-width: 450px) {
+@media (max-width: 725px) {
   .incident-date {
     width: 100%;
   }
+  .incident-title {
+    width: 100%;
+  }
 }
 
 .incident-description {
   font-size: 14px;
   font-weight: 400;
 }
 
+.separate {
+  text-align: right;
+  padding-bottom: 1em;
+  border-bottom: 2px solid #F7F7F8;
+}
+
 .incident-status {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
   margin: 24px 0 0 0;
 }
```

Comment:
Since the rules added here are identical to the rules in the `flex-out` class, I'm wondering why you didn't add the  class to the element instead?

Reply:
good catch, i thought there might be another one of those floating around.